pages/linux/qm-importdisk.md

```diff
@@ -0,0 +1,13 @@
+# qm importdisk
+
+> Import disk image to a VM as unused disk.
+> The image format has to be supported by qemu-img(1) such as raw, qcow2, qed, vdi, vmdk, vhd.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Import a VMDK disk image to storage called "local-zfs":
+
+`qm importdisk {{100}} name-of-disk.vmdk local-zfs`
```

Comment:
```suggestion
`qm importdisk {{vm_id}} {{disk.vmdk}} {{local-zfs}}`
```

Reply:
This command requires the path to the vmdk file, right? Is an storage arbitrary name?
```suggestion
`qm importdisk {{100}} {{path/to/disk.vmdk}} {{storage_name}}`
```